action.yml

```diff
@@ -36,11 +36,19 @@ inputs:
   settings-path:
     description: 'Path to where the settings.xml file will be written. Default is ~/.m2.'
     required: false
+  gpg-private-key:
+    description: 'GPG private key to import. Default is empty string.'
+    required: false
+  gpg-passphrase:
+    description: 'Environment variable name for the GPG private key passphrase. Default is
+       $GPG_PASSPHRASE.'
+    required: false
 outputs:
   path:
     description: 'Path to where the java environment has been installed (same as $JAVA_HOME)'
   version:
     description: 'Actual version of the java environment that has been installed'
 runs:
   using: 'node12'
-  main: 'dist/index.js'
+  main: 'dist/setup/index.js'
```

Comment:
I like this approach of separate files. 👍

`checkout` has a `post` step as well, but it lumps everything together into a single file. I think what you have is better. https://github.com/actions/checkout/blob/592cf69a223b04e75ddf345919130b91010eb2a6/action.yml#L73